src/main/java/seedu/smartnus/model/statistic/Statistic.java

```diff
@@ -85,6 +85,14 @@ public int getCorrectPercentage() {
         return (int) (correct / attempt * 100);
     }
 
+    /**
+     * Returns if the statistic is valid.
+     * @return If the statistic is valid.
+     */
+    public boolean isValidStatistic() {
+        return correctCount <= attemptCount;
```

Comment:
should this also check that `correctCount` is >= 0 to prevent any negatives?

Once `correctCount` is non-negative, `attemptCount` will also follow as it >= `correctCount`

Reply:
Hmm not even sure how `correctCount` can be less than 0 but sure.
Changed to ensure that both `attemptCount` and `correctCount` are not negative